app/helpers/state_file/nj_2450_helper.rb

```diff
@@ -0,0 +1,13 @@
+module StateFile
+  module Nj2450Helper
+    def line_name(line, primary_or_spouse)
+      return :"#{line}_SPOUSE" if primary_or_spouse == :spouse
+      :"#{line}_PRIMARY"
+    end
+
+    def get_persons_w2s(intake, primary_or_spouse)
+      ssn = primary_or_spouse == :primary ? intake.primary.ssn : intake.spouse.ssn
+      intake.state_file_w2s.all&.select { |w2| w2.employee_ssn == ssn }
+    end
```

Comment:
[dust|pebble] if there are only two valid inputs for `primary_or_spouse`, can it be turned into a `boolean`?